compose/gn3_postgis/docker-compose.yml

```diff
@@ -9,23 +9,19 @@ version: '2'
 services:
 
     geonetwork:
-      image: geonetwork:3.0-postgres
+      build:  ./geonetwork
       ports:
           - 8080:8080
       environment:
           DATA_DIR: /var/lib/geonetwork_data
           POSTGRES_DB_USERNAME: postgres
           POSTGRES_DB_PASSWORD: mysecretpassword
-      volumes:
-          - "/host/geonetwork-docker:/var/lib/geonetwork_data"
       links:
-          - postgis:postgres
+          - postgres
       depends_on:
-          - postgis
+          - postgres
     
-    postgis:
```

Comment:
GN expects to link to a container called "postgres". Either "postgis" needs to be renamed "postgres", or we have to relabel it on the link (as I had in the previous commit).